exec/java-exec/src/main/java/org/apache/drill/exec/planner/PlannerPhase.java

```diff
@@ -346,6 +355,20 @@ static RuleSet getPruneScanRules(OptimizerRulesContext optimizerRulesContext) {
   }
 
   /**
+   *   Get an immutable list of partition pruning rules that will be used in logical planning.
+   */
+  static RuleSet getPhysicalPruneScanRules(OptimizerRulesContext optimizerRulesContext) {
+    final ImmutableSet<RelOptRule> pruneRules = ImmutableSet.<RelOptRule>builder()
+        .add(
+            ParquetPushDownFilter.getFilterOnProject(optimizerRulesContext),
```

Comment:
Opened DRILL-4998, and put comments to explain why we put this rule in physical planning phase.